packages/pds/tests/views/admin/invites.test.ts

```diff
@@ -199,4 +217,39 @@ describe('pds admin invite views', () => {
     )
     await expect(attemptCreateInvite).rejects.toThrow('Authentication Required')
   })
+
+  it('disables an account from getting additional invite codes', async () => {
+    await agent.api.com.atproto.admin.disableAccountInvites(
+      { account: carol },
+      { encoding: 'application/json', headers: { authorization: adminAuth() } },
+    )
+
+    // does not disable existing invites
```

Comment:
This comment might be out of date, or maybe I'm just totally understanding it (or maybe there's a bug?).  I would expect there to be some codes in the response below in order to show that existing invite codes are still enabled.